src/components/Nav.js

```diff
@@ -2,11 +2,11 @@ import React from "react";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faMusic } from "@fortawesome/free-solid-svg-icons";
 
-export const Nav = ({ libraryStatus, setLibraryStatus }) => {
+export const Nav = ({ toggleLibraryStatus }) => {
```

Comment:
Tutaj trochę może trochę mojej nadgorliwości, ale tak, jeśli libraryStatus jest potrzebny tylko po to, żeby go zanegować w swoim setterze, to można zrobić toggle i nie zawrać komponentowi głowy propsami, które nie są niezbędne. Takie coś oczywiście jest bez sensu, jeśli chciałbyś np. zrobić różne ikony dla otwartego/zamkniętego library.